kubernetes/skaffold/src/main/kotlin/com/google/kubernetes/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -119,7 +121,13 @@ abstract class AbstractSkaffoldRunConfiguration(
 
     override fun checkConfiguration() {
         if (!SkaffoldExecutorService.instance.isSkaffoldAvailable()) {
-            throw RuntimeConfigurationWarning(message("skaffold.run.config.not.on.system.error"))
+            throw RuntimeConfigurationWarning(message("skaffold.run.config.not.on.system.error"),
+                    Runnable {
```

Comment:
Oh that's how they implement this nice `fix it` action?

Reply:
yep, its pretty convenient